README.md

```diff
@@ -1,5 +1,26 @@
 # audit-marketplace
 
+## Change record
+
+- feat: add marketitemcount & auctionitemcount (#1)
+
+> - Add marketitemcount and auctionitemcount functions.
+
+- feat: add code comment (#3)
+
+> - Add code comments.
+
+- Feature/marketplace (#4)
+
+> - Add some checks.
+> - Modify the MarketItem and AuctionItem data structures to add soldOut boolean parameters to them.
+> - Change all list functions to reverse.
+> - Modifying the remove related functions to no longer delete item, but adjusts the item parameter soldOut to true.
```

Comment:
The remove-related functions now sets soldOut field to true instead of delete items permantly.